SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialStatics, Log, All);
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
+public:
+	/**
+	 * Returns whether Spatial networking has been enabled or not */
+	UFUNCTION(BlueprintPure, Category = "Game")
+	static bool SpatialNetworkingEnabled();
+
+	/**
+	 * Returns whether offloading has been enabled or not, returns false if spatial networking is disabled */
+	UFUNCTION(BlueprintPure, Category = "Game")
+	static bool SpatialOffloadingEnabled();
+
+	/**
+	 * Returns true without SpatialOS offloading enabled, or if offloading is enabled, returns true is actor group
+	 * worker association matches the current worker type. */
+	UFUNCTION(BlueprintPure, Category = "Game")
+	static bool ActorGroupAssociationMatchesWorkerType(const AActor* Actor);
+
+	/**
+	 * Returns whether the game instance for the World has the UnrealWorker type */
+	UFUNCTION(BlueprintPure, Category = "Game")
+	static bool IsGameServer(const UWorld* World);
```

Comment:
This seems like an ambiguous name. Without reading the comment I don't know what it does and even then I'm not 100%.

Reply:
@samiwh is this with respect to original code or Sahil's comment? was planning on changing to Sahils comment?